api/src/main/java/edu/cornell/mannlib/vitro/webapp/dao/WebappDaoFactoryConfig.java

```diff
@@ -5,26 +5,37 @@
 import java.util.Arrays;
 import java.util.HashSet;
 import java.util.List;
+import java.util.Locale;
 import java.util.Map;
 import java.util.Set;
 
 import edu.cornell.mannlib.vitro.webapp.dao.PropertyDao.FullPropertyKey;
 
 public class WebappDaoFactoryConfig {
 
+	private List<Locale> preferredLocales;
 	private List<String> preferredLanguages;
 	private String defaultNamespace;
 	private Set<String> nonUserNamespaces;
 	private boolean isUnderlyingStoreReasoned = false;
 	public Map<FullPropertyKey, String> customListViewConfigFileMap;
 
 	public WebappDaoFactoryConfig() {
-	    preferredLanguages = Arrays.asList("en-US", "en", "EN");
+		preferredLocales = Arrays.asList(Locale.getDefault());
+		preferredLanguages = Arrays.asList("en-US", "en", "EN");
```

Comment:
Would it make more sense to use default locale language rather than assuming English as default?

Reply:
@wwelling : Is there any guarantee that the default locale language will be included in the list of available languages?

Reply:
Doesn't seem to be any guarantee there. That is why one should define the other. Should we create a default locale that would match the hard-coded English preferred languages? Or should we create default preferred languages from the system default locale?

Reply:
I'm not sure Locale.default() is of much value here.  It's one thing to use it for formatting dates and currency on a page, but there's a very good chance that the default locale will not match a language VIVO has available.  I'm pretty sure my JVMs' default locales are ro_RO, which isn't going to match anything in VIVO.

Reply:
Ok. I am going to create preferred locales from preferred languages. This will ensure they match.